src/actions/transferActions.js

```diff
@@ -692,17 +692,63 @@ async function _getTransferHistory () {
   let transferData = await API.getBatchTransfers({ sendingId: sendingIds })
   transferData = transferData.map(item => {
     let state = null
-    if (!item.receiveTxHash) {
-      if (!item.cancelTxHash) {
-        // pending receive
-        state = 'pending'
-      } else {
-        // cancelled
-        state = 'cancelled'
+    const { sendTxState, receiveTxState, cancelTxState } = item
+    switch (sendTxState) {
+      case 'Pending': {
+        // SEND_PENDING
+        state = 'Sending'
```

Comment:
export a list of transfer states for other components to use:

```js
const TRANSFER_STATES = {
  SEND_CONFIRMED_RECEIVE_PENDING: 'SEND_CONFIRMED_RECEIVE_PENDING',
  ...
}
```
Make sure to export the constants

return the corresponding state constant instead of string, i.e.
```js
return TRANSFER_STATES.SEND_CONFIRMED_RECEIVE_PENDING
```